deepcell/utils/backbone_utils.py

```diff
@@ -465,8 +465,10 @@ class Utils(object):  # pylint: disable=useless-object-inheritance
 
         time_distributed_outputs = []
         for i, out in enumerate(layer_outputs):
+            td_name = 'td{}'.format(i)
+            md_name = 'md{}'.format(i)
```

Comment:
I like `td` shorthand, but I'm not sure about `md`. I don't think there's any harm in having a more descriptive name.

Reply:
done